### ISSUE TITLE
Check if and how EKS clusters will be related to Organizations and RBAC permissions

### ISSUE THREAD

**Issue by piontec** [giantswarm/roadmap#2669]
*Note: [The EKS as CAPI task](https://github.com/giantswarm/roadmap/issues/2668) has to be completed to unlock this issue*

As we agreed, we will keep `Organizations` concept for the MC that is expected to read-only import existing vanilla EKS clusters. This means, that a secret containing `kube.config` and CAPI object definition (most probably `Cluster` CR) can be placed within a scope of an organization and will work as expected in terms of RBAC access.

So, the scope of this ticket is to evaluate the following on the `snail` PoC MC:
- [x] create test Organizations on the MC
- [ ] put the existing EKS workload cluster access objects (`kube.config` Secret, `Cluster` CR) into the organization scope and confirm, that access permissions to that EKS work as expected.

**Comment by gawertm** [comment]
hi @piontec, we discussed this a bit during our onside.
generally we do not see any issues with doing that. Maybe you can say where your concerns come from? Maybe we are missing some information from the EKS adoption epic.
we do not reconcile any ressources inside the organizations. only the organization CRs and org namespaces themselves

**Comment by gawertm** [comment]
Ok so I guess we can go ahead and try it out. When do you expect the other issue mentioned above to be resolved ? Just for our planning

**Comment by mproffitt** [comment]
@gawertm This should be ready for testing as https://github.com/giantswarm/roadmap/issues/2668 was closed off today

**Comment by gawertm** [comment]
hi, we discussed this during standup and probably @anvddriesch will reach out to you after finishing her current task :) Expected maybe even next week already

**Comment by anvddriesch** [comment]
Notes from looking at the test installation.
- as expected, creating an organization works just fine
- i have not replicated the cluster import for testing. Is there a guide for it? The following is based on already existing imported eks cluster that was present in the installation
- CAPI resources are within the organization namespace (Cluster, AWSManagedControlplane, etc) as usual so there is no difference in terms of access
- logging into the eks cluster requires the user to have access to the AWSManagedControlplane and a kubeconfig secret which are inside the org namespace, so it's similar to other installations so people with access to the org won't have an issue.
- Crossplane resources related to the Cluster (like VPC, Nodegroup) are not namespaced and therefore exist outside the organization. People with org access do not automatically get access to these resources related to imported eks clusters within that org
- installation-wide customer admin groups however do have access to these resources since its part of the `crossplane-edit` role which is automatically bound by rbac-operator

So basically, if we want people with org access to also have access to crossplane resources related to clusters within the org, we need to extend rbac-operator.

**Comment by gawertm** [comment]
@piontec @mproffitt can you provide feedback on the above? is that enough for you? or do you need us to test also importing a cluster? are we ok, that other crossplane resources are outside of the organization?

**Comment by piontec** [comment]
I think the above is good enough, at least to get started. We want people in orgs to have access to ECs, they don't have know/control the infrastructure that provides that. Thanks for checking! If we need more, we ping you again - closing for now.
